mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -79,6 +79,49 @@ func resourceContainerNodePool() *schema.Resource {
 	}
 }
 
+var schemaBlueGreenSettings = &schema.Schema{
+	Type:     schema.TypeList,
+	Optional: true,
+	Computed: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"standard_rollout_policy": {
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: `Standard rollout policy is the default policy for blue-green.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"batch_percentage": {
+							Type:         schema.TypeFloat,
+							Optional:     true,
+							Description:  `Percentage of the blue pool nodes to drain in a batch.`,
+							ValidateFunc: validation.FloatBetween(0.0, 1.0),
+						},
+						"batch_node_count": {
```

Comment:
Will the API return values for these two fields if they are not explicitly specified by users. If so, we need to added `computed: true` to these fields.

Reply:
Whoops, just realize that I was leaving unfinished comments in the previous one. What I wanted to say was "If these are not explicitly specified by users, but the API still return values for them, please add optional + computed". Sorry for the misleading!